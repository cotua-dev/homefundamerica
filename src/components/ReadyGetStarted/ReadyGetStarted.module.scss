section#readyGetStartedSection {
    height: 30vh;

    @media screen and (max-width: 950px) {
        text-align: center;
        height: 100%;
    }
}
