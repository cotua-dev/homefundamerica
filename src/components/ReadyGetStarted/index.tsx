import React, {
    FunctionComponent,
    ReactElement,
} from 'react';
import { moveToStepper } from '../../utils/moveToStepper';
import { readyGetStartedSection } from './ReadyGetStarted.module.scss';
import * as sharedStyles from '../sharedStyles/shared.module.scss';

export const ReadyGetStarted: FunctionComponent = (): ReactElement => {
    return (
        <section id={readyGetStartedSection} className={sharedStyles.loanPositive}>
            <div className={sharedStyles.contentWrapper}>
                <h2>{`Ready To Get Started?`}</h2>
                <p>
                    {`
                        It's easy to apply, just answer some basic question
                        about your personal finances and the home.
                    `}
                </p>
                <button
                    className={sharedStyles.applyButton}
                    type="button"
                    onClick={moveToStepper}
                >Apply Now</button>
            </div>
        </section>
    );
}
