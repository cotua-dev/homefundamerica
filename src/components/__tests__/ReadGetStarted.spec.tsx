import React from 'react';
import {
    render,
    RenderResult,
} from '@testing-library/react';
import { ReadyGetStarted } from '../ReadyGetStarted';

describe('ReadyGetStarted Component', (): void => {
    test('renders', (): void => {
        const { container }: RenderResult = render(<ReadyGetStarted/>);
        expect(container.firstChild).toMatchSnapshot();
    });
});
