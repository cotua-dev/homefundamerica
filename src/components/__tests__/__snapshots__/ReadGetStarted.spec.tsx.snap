// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReadyGetStarted Component renders 1`] = `
<section
  id="readyGetStartedSection"
>
  <div>
    <h2>
      Ready To Get Started?
    </h2>
    <p>
      
                        It's easy to apply, just answer some basic question
                        about your personal finances and the home.
                    
    </p>
    <button
      type="button"
    >
      Apply Now
    </button>
  </div>
</section>
`;
